pkg/pillar/pubsub/subscribe.go

```diff
@@ -108,7 +108,7 @@ func (sub *SubscriptionImpl) populate() {
 //   fooAll := s1.GetAll()
 func (sub *SubscriptionImpl) ProcessChange(change Change) {
 	start := time.Now()
-	sub.log.Debugf("ProcessChange agentName(%s) agentScope(%s) topic(%s): %#v", sub.agentName, sub.agentScope, sub.topic, change)
+	sub.log.Debugf("ProcessChange agentName(%s) agentScope(%s) topic(%s): %s", sub.agentName, sub.agentScope, sub.topic, change)
```

Comment:
Oh, The original code was leaking info here as well. So we clearly need to change it.

Reply:
I can see it only with debug mode (I ran (zed)client with `-d` flag in my experiments with issue about RPi). So, maybe it is not a problem.

Reply:
One can remotely set the loglevel to trace and get those messages logged. That is why there are comments in the code to not ever log the content of the objects.